apps/admin-gui-e2e/src/integration/vo-management-perun_admin.spec.js

```diff
@@ -1,6 +1,11 @@
 /// <reference types="cypress" />
 
 context('Actions', () => {
+  const dbVoName = 'test-e2e-vo-from-db';
+  const dbGroupName = 'test-group-from-db';
+  const dbApplicationItemTextFieldName = 'input-test';
+  const dbVoManager = 'vo-manager';
+  const dbUser = 'test';
 
   before(() => {
     if (Cypress.env('BA_USERNAME')) {
```

Comment:
I have realized one more thing. I think we wanted to do this test with a different principal - a user with only voAdmin role. Could you please implement it? It also requires the user to be created in the DB with the appropriate role.